tools/kill_cleanup_disttrain.sh

```diff
@@ -12,10 +12,16 @@ done < "$1"
 
 graph_name=$2
 
+if [ -z "$3"]; then
+    port_num=2222
```

Comment:
port_num=22?
This is the default port number for ssh. I believe most of people assume that by default ssh uses 22.

Reply:
This command should be executed within our Docker container. So the default should be 2222 as we designed.

If use 22, the command will fail and users have no clue what port number to use.